tools/bin/cool_python_thing.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/python3
+
+import os
+import sys
+
+# get current directory
+path = os.getcwd()
+print(f"Current Directory is {path}")
+
+# prints parent directory
+parent_dir = os.path.abspath(os.path.join(path, os.pardir))
+print(f"parent Directory is {parent_dir}")
+
+# prints module path
+module_path = os.path.abspath(os.path.join(parent_dir, "python_lib"))
+print(f"module path is {module_path}")
+
+
+sys.path.append(module_path)
+
+
+from immma_module import Topher_class as Better_named_class
```

Comment:
this is the important part, the rest is for teaching

Reply:
this method adds `python_lib` directory to the executable paths of python.  This allow for import without any further pathing as python will go through all files in the known paths to look for `immma_module`